frontend/packages/text-editor/src/RightMenu.tsx

```diff
@@ -70,22 +69,18 @@ export const RightMenu = ({
                       confirmText={t('schema_editor.language_confirm_deletion')}
                       onConfirm={() => handleDeleteLanguage(langCode)}
                       onClose={() => setLangCodeToDelete(undefined)}
-                      trigger={
-                        <StudioButton
-                          variant={canDeleteLang(langCode) ? 'primary' : 'secondary'}
-                          data-testid={deleteButtonId(langCode)}
-                          color='danger'
-                          onClick={() =>
-                            setLangCodeToDelete((prevState) =>
-                              prevState === langCode ? undefined : langCode,
-                            )
-                          }
-                          disabled={!canDeleteLang(langCode)}
-                          aria-label={t('schema_editor.language_delete_button')}
-                        >
-                          {t('schema_editor.language_delete_button')}
-                        </StudioButton>
-                      }
+                      triggerProps={{
+                        variant: canDeleteLang(langCode) ? 'primary' : 'secondary',
+                        'data-testid': deleteButtonId(langCode),
+                        color: 'danger',
+                        onClick: () =>
+                          setLangCodeToDelete((prevState) =>
+                            prevState === langCode ? undefined : langCode,
+                          ),
+                        disabled: !canDeleteLang(langCode),
+                        'aria-label': t('schema_editor.language_delete_button'),
```

Comment:
Regarding my previous comment, I guess it has something to do with this 😉
But then I do not understand why 'aria-label' is accepted by default?

Reply:
Aria attributes are included in the definition of the `HTMLAttributes` type, while data attributes are not.

Reply:
Nice, thanks for the explanation!